### PR TITLE
Bug 1050417 - Remove One and Done User REST API

### PR DESCRIPTION
**Pull request by bobsilverberg**

### CODE REVIEW

**Comment by bobsilverberg** [comment]
@akatsoulas For a number of reasons we have decided to remove the Users API, but we're going to keep the Task API for now.

r?

File: oneanddone/users/views.py [hunk 1]
```diff
@@ -1,27 +1,25 @@
...
 from django.http import Http404
 from django.shortcuts import redirect
 from django.views import generic
 
-from rest_framework import generics
 from braces.views import LoginRequiredMixin
 import django_browserid.views
-from tower import ugettext as _
 from random import randint
 import re
```
Comment (akatsoulas) [inline oneanddone/users/views.py]: Personally, I prefer all the `import` statements before `from`.

**Comment by akatsoulas** [comment]
Apart from the minor comment, everything looks good here. r+ :rocket:

**Comment by bobsilverberg** [comment]
Thanks for the review @akatsoulas!

**Pull request merged**